packages/client/hmi-client/src/services/model.ts

```diff
@@ -74,21 +72,13 @@ export async function reconstructAMR(amr: any) {
 }
 
 // function adds model to project, returns modelId if successful otherwise null
-export async function addNewModelToProject(
-	modelName: string,
-	project: IProject
-): Promise<string | null> {
+export async function addNewModelToProject(modelName: string): Promise<string | null> {
```

Comment:
Where is the part which we add the model to the project?

Reply:
Found it in the composable `model.ts`. It calls this as its step 1

Reply:
This ain't pretty at all.

Reply:
In my opinion this function should not even be in `model.ts`, it should be in `tera-model-modal` where it is being used.

Currently `tera-modal-modal` is calling `model.ts`, which calls 2 other services (petrinet service and project service) inside of it. I think `tera-modal-model` should do something like 

```
PetrinetService.newAmr();
ModelService.createModel();
useProjects().addAsset();
```
That way we limit the side effects hidden inside method calls.